### PR TITLE
Fixed bug where the versioning info of schemas was lost

### PR DESCRIPTION
**Pull request by rasmus-kirk**
## Purpose

See #173

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/nodejs/test/clientV2.test.ts [hunk 1]
```diff
@@ -341,9 +341,12 @@ test.each([clientV2, clientWeb])('getModuleSource', async (client) => {
     );
 
     const localModuleHex = Buffer.from(localModuleBytes);
-    const moduleSource = await client.getModuleSource(moduleRef, testBlockHash);
+    const versionedModuleSource = await client.getModuleSource(
+        moduleRef,
+        testBlockHash
+    );
 
-    expect(localModuleHex).toEqual(moduleSource);
```
Comment (soerenbf) [inline packages/nodejs/test/clientV2.test.ts:346]: Maybe add a test that the expected version is returned as well?

**Review by soerenbf — APPROVED** [review]
Looks good :)

File: packages/common/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+### Breaking changes
+- Fixed bug in `getModuleSchema`. It now returns a schema with version.
```
Comment (shjortConcordium) [inline packages/common/CHANGELOG.md:6]:
```suggestion
- Added missing version return type in `getModuleSchema`. It now returns an object containing the schema source and version.
```

Comment (shjortConcordium) [reply packages/common/CHANGELOG.md:6]: Making it more clear. :thinking:

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**